test/state/precompiles.cpp

```diff
@@ -363,11 +366,24 @@ ExecutionResult expmod_execute(
     const auto exp = payload.substr(base_len, exp_len);
     const auto mod = mod_requires_padding ? bytes_view{output, mod_len} : mod_explicit;
 
-#ifdef EVMONE_PRECOMPILES_GMP
+    uint8_t evmone_output[1024];  // Use a separate buffer not to overwrite the truncated mod.
+    const auto in_range = crypto::modexp(base, exp, mod, evmone_output);
+    if (in_range)
+    {
+        uint8_t gmp_output[1024];
+        expmod_gmp(base, exp, mod, gmp_output);
+        if (std::memcmp(evmone_output, gmp_output, mod_len) != 0)
```

Comment:
This is a temporary code for fuzzing. Maybe just move it under `#ifndef NDEBUG` (fuzzer compiles with asserts enabled).